tests/python/topi/python/test_topi_prng.py

```diff
@@ -157,8 +185,27 @@ def test_uniform(target, dev):
         assert np.max(rands) <= 10.0
 
 
+@tvm.testing.parametrize_targets
+def test_multinomial(target, dev):
+    def _verify_multinomial(size, num_samples):
```

Comment:
Could you do some rough checking of expected value and variance of the distribution. It's always hard to tell if these random things are implemented correctly, but I think this would help.

Reply:
Is there a good way to do this without potentially introducing flakiness? I guess we could use a fixed seed. Would that be satisfactory?

Reply:
You could generate a "large" sample of at least 10,000 values and then use a chi-squared test (scipy.stats.chisquare). You'd look at the p-value from that chi-squared test and compare it to an acceptably low threshold for flakiness -- for example, have this unit test fail if the p-value is smaller than 1e-6, which should only happen by chance in one run per million.)

Reply:
Thanks for this tip. I added a chisquare test which confirms that the behavior of this function is expected.